README.rst

```diff
@@ -29,38 +48,25 @@ Installation
         ftw.linkchecker
 
 
-One needs to add a config file (e.g. linkchecker_config.json) holding:
+Configuration
+=============
+
+A config file is required (there is an example in ``config.example``).
+In the config file following things can be configured:
```

Comment:
```suggestion
The following things can be configured in the config file:
```

Reply:
Yup, but 'things' should be 'options' too .  (language note: 'things' sounds too unspecific and less professional)